README.md

```diff
@@ -186,18 +186,19 @@ See the file `licenses/commons-math.txt` for the copyright notices.
 See the file `licenses/Apache-2.0.txt` for the full license text.
 See the file `licenses/fast-util.txt` for the copyright notice.
 
-[1]: http://chunky.llbit.se
-[2]: http://chunky.llbit.se/troubleshooting.html
+[1]: https://chunky-dev.github.io/docs/
+[2]: https://chunky-dev.github.io/docs/faq/troubleshooting/
 [3]: http://www.reddit.com/r/chunky
-[4]: http://llbit.se
-[5]: http://chunky.llbit.se/headless.html
+

```

Comment:
Maybe we should stop using numbers and use IDs instead, like `[chunky-docs]: https://chunky-dev.github.io/docs`?

Reply:
That might be easier and more editor friendly. I'll work on that shortly

Reply:
Addressed in a0388c8